ts/screens/ingress/IngressScreen.tsx

```diff
@@ -1,104 +1,63 @@
-import { IOStyles } from "../../components/core/variables/IOStyles";
-import SectionStatusComponent from "../../components/SectionStatus";
-import I18n from "../../i18n";
-import { ReduxProps } from "../../store/actions/types";
+import { View, StyleSheet, AccessibilityInfo, Platform } from "react-native";
 import {
-  sessionInfoSelector,
-  sessionTokenSelector
-} from "../../store/reducers/authentication";
-import { profileSelector } from "../../store/reducers/profile";
-import { GlobalState } from "../../store/reducers/types";
-import variables from "../../theme/variables";
-import { IngressCheckBox } from "./CheckBox";
-
-type Props = ReduxProps & ReturnType<typeof mapStateToProps>;
+  ContentWrapper,
+  H3,
+  IOStyles,
+  LoadingSpinner,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { SafeAreaView } from "react-native-safe-area-context";
+import I18n from "../../i18n";
+import { useOnFirstRender } from "../../utils/hooks/useOnFirstRender";
+import { trackIngressScreen } from "../profile/analytics";
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
-    paddingTop: variables.contentPaddingLarge,
-    backgroundColor: variables.brandPrimary
+    ...IOStyles.bgWhite,
+    ...IOStyles.centerJustified,
+    ...IOStyles.flex
   }
 });
 
-class IngressScreen extends React.PureComponent<Props> {
-  public render() {
-    const items = [
-      {
-        enabled: this.props.hasSessionToken,
-        label: I18n.t("startup.authentication")
-      },
-      {
-        enabled: this.props.hasSessionInfo,
-        label: I18n.t("startup.sessionInfo")
-      },
-      { enabled: this.props.hasProfile, label: I18n.t("startup.profileInfo") },
-      {
-        enabled: this.props.isProfileEnabled,
-        label: I18n.t("startup.profileEnabled")
-      }
-    ];
-    return (
-      <SafeAreaView
-        style={[IOStyles.flex, { backgroundColor: variables.brandPrimary }]}
-      >
-        <StatusBar
-          barStyle="light-content"
-          backgroundColor={styles.container.backgroundColor}
-        />
-        <Container style={styles.container}>
-          <View style={IOStyles.alignCenter}>
-            <Body color="white">{I18n.t("startup.title")}</Body>
-          </View>
-          <Spinner color="white" />
-
-          <List withContentLateralPadding={true}>
-            {items.map((item, index) => (
-              <ListItem key={`item-${index}`}>
-                <IngressCheckBox checked={item.enabled} />
-                <HSpacer size={8} />
-                <Body
-                  color="white"
-                  weight={item.enabled ? "SemiBold" : "Regular"}
-                >
-                  {item.label}
-                </Body>
-              </ListItem>
-            ))}
-          </List>
-          <View style={{ marginTop: 48 }}>
-            <SectionStatusComponent sectionKey={"ingress"} />
-          </View>
-        </Container>
-      </SafeAreaView>
-    );
-  }
-}
+const SPINNER_SIZE = 48;
```

Comment:
Is this value only used for this screen or is a generic rule so we can extract in a global definition?

Reply:
Good question. This let me be aware that the component to use is not the one from the DS directly but the `LoadingIndicator` [that wraps it with the correct styling](https://pagopaspa.slack.com/archives/C013V764P9U/p1707928510151789).